packages/design/scripts/generateIconMap.ts

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ts-node  --project ../../tsconfig.bin.json
+
+import { readFileSync, readdirSync, statSync, writeFileSync } from "fs";
+import { join, parse } from "path";
+import { load } from "cheerio";
+
+const iconFolder = join(__dirname, "../assets/icons");
+const iconMap: { height: number; icons: Record<string, string[]> } = {
+  height: 720,
```

Comment:
Great work on this PR - makes the process muuuuch simpler.
I'm still confused about this height though - why and why 720 now 😆

Reply:
that actually needs to match the SVG sizes. if you look at the source code on one of them 
https://github.com/GetJobber/atlantis/blob/2346a7644c4f24c260835a871e77e996bd8e0201/packages/design/assets/icons/invoice.svg?short_path=fab3131#L2

it has a width and height of 720. 

It should just be 24 tho since that's what we use in figma. Will update it soon here since I have the original SVG's now

Reply:
Got it - I saw the 720 in the source code but wasn't sure what came first - the svg or if we were telling the svg to have that in the source code. Makes sense that we'll change it to 24 to match Figma 👍